### PR TITLE
Fixes Codeblock Language With Only Rustdoc Flags

### PR DESCRIPTION
**Pull request by zicklag**
Another quick fix. This makes sure that codeblocks that only have special rustdoc flags such as `no_run` have the `rust` lang applied when adding to the README.

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -81,8 +83,14 @@ pub fn emit(input: InputFile, template: &str, out_file: &mut dyn io::Write) -> a
...
 				},
 				Tag::CodeBlock(CodeBlockKind::Fenced(lang)) => {
-					let lang: &str = &lang;
-					let lang = match lang {
+					// Strip rustdoc code-block flags from the language
+					let mut lang = lang.to_string();
+					for flag in RUSTDOC_CODEBLOCK_FLAGS {
+						lang = lang.replace(flag, "");
+					}
+					lang = lang.replace(",", "");
```
Comment (msrd0) [inline src/output.rs:91]: This code removes all `,` from the `lang` string, and the next match statement checks for `rust,` appearing in the same string. I assume the next statement can be stripped down to `if lang.is_empty() { lang = "rust".to_owned(); }`?

**Review by msrd0 — COMMENTED** [review]
Thanks! I didn't even know you could write `no_run` instead of `rust,no_run`

File: src/output.rs (hunk 1 above)
Comment (zicklag) [reply src/output.rs:91]: Ah, good point.

Comment (zicklag) [reply src/output.rs:91]: Pushed an update.

**Review by msrd0 — APPROVED** [review]
Thanks!

**Pull request merged**